package.json

```diff
@@ -63,6 +63,8 @@
     "mini-css-extract-plugin": "^0.5.0",
     "minimist": "^1.2.5",
     "mocha": "^4.0.1",
+    "moment": "^2.29.1",
```

Comment:
Do we really need this package to be added separately because i don't think there is any direct usage of this package in the PR.

Reply:
It's a prerequisite for it.